src/ServicePulse.Host/app/js/views/archive/controller.js

```diff
@@ -72,15 +73,29 @@
             vm.loadingData = false;
         };
 
+        var storeGroupOption = function (amount, unit) {
+            $cookies.put('archive_amount', amount);
+            $cookies.put('archive_unit', unit);
+        }
+
+        var loadDefaultGroup = function() {
```

Comment:
This isn't loading a default group, this is loading the group from the cookies. Maybe `loadInitialGroupFromCookies`?

Also, what is the result if there aren't cookies here? What is the default behaviour?

Reply:
I also kind of feel like this should be separated into 2 methods:

1 to load the values from cookies (or provide a default)
2 to save the values in the `timeGroup` variable

Reply:
Maybe since it's a new function, switch to arrow function?

```
var loadDefaultGroup = () => {
};
```

Reply:
It does provide the default. The default, if the cookies are not available is `undefined` which would automatically default the filter to 'All Archived' messages.